src/operator/leaky_relu-inl.h

```diff
@@ -217,7 +237,7 @@ class LeakyReLUOp : public Operator {
 };  // class LeakyReLUOp
 
 template<typename xpu>
-Operator* CreateOp(LeakyReLUParam type);
+Operator* CreateOp(LeakyReLUParam type, int dtype);
```

Comment:
Sorry, I'm not familiar with the cpp code, but is there no typedef for dtype - int is kind of arbitrary.

Reply:
The types are represented by integers, please see leaky_relu.cc:43 for how this function is being called. Also you can refer to https://github.com/dmlc/mshadow/blob/b3771de20ed36f90ba7b8436ae4b79ea298a687a/mshadow/base.h#L288 for more info about how each integer map to a type. This change is written in a similar way as #2322 #3011 #10078 #10125 .

Reply:
Great, thanks for the explanation :)

Reply:
My pleasure! 😄